src/EasyAbp.Abp.SettingUi.Web/Pages/SettingUiPageContributor.cs

```diff
@@ -13,6 +11,11 @@ public class SettingUiPageContributor : ISettingPageContributor
     {
         public async Task ConfigureAsync(SettingPageCreationContext context)
         {
+            if (!await CheckPermissionsAsync(context))
```

Comment:
I think calling `CheckPermissionAsync` is not nesseary, 
since it has already been called [here](https://github.com/abpframework/abp/blob/dev/modules/setting-management/src/Volo.Abp.SettingManagement.Web/Navigation/SettingManagementMainMenuContributor.cs#L56)

Reply:
The method `CheckAnyOfPagePermissionsGranted` is only for SettingManagementMainMenu.
When the user has any SettingsContributor permissions,setting management menu will be displayed.

If the use don’t have SettingUi permission but he have any other SettingsContributor permissions, user access the setting management page will cause an exception `No SettingUi policy granted`.

Reply:
[EmailingPageContributor](https://github.com/abpframework/abp/blob/dev/modules/setting-management/src/Volo.Abp.SettingManagement.Web/Settings/EmailingPageContributor.cs) is also dot it.